--- sim5lib.py.orig	2021-03-20 19:25:38.954656044 +0100
+++ sim5lib.py	2021-03-20 19:27:29.469811216 +0100
@@ -8,11 +8,17 @@
 if _swig_python_version_info < (2, 7, 0):
     raise RuntimeError("Python 2.7 or later required")
 
+# append the path to shared-object module.
+import os,sys
+sim5dir = os.path.normpath(os.path.dirname(os.path.abspath(__file__))+'/../lib')
+sys.path.append(sim5dir)
+import sim5lib as _sim5lib
+
 # Import the low-level C/C++ module
-if __package__ or "." in __name__:
-    from . import _sim5lib
-else:
-    import _sim5lib
+#if __package__ or "." in __name__:
+#    from . import _sim5lib
+#else:
+#    import _sim5lib
 
 try:
     import builtins as __builtin__
